scripts/us_census/pep/monthly_population_estimate/download.py

```diff
@@ -0,0 +1,243 @@
+    # Table 1. Monthly Population Estimates for the United States:  April 1, 2000 to December 1, 2010
+    # Year and Month    Resident Population     Resident Population Plus Armed Forces Overseas   Civilian Population	Civilian Noninstitutionalized Population
+    # 2000
+    # .April 1	28,14,24,602	28,16,52,670	28,02,00,922	27,61,62,490
+    # .May 1	28,16,46,806	28,18,76,634	28,04,28,534	27,63,89,920
+    #
+    # Output Data:
+    # (Made Headers) Year and Month    Resident Population     Resident Population Plus Armed Forces Overseas   Civilian Population    Civilian Noninstitutionalized Population
+    # 2000
+    # .April 1	28,14,24,602	28,16,52,670	28,02,00,922	27,61,62,490
+    # .May 1	28,16,46,806	28,18,76,634	28,04,28,534	27,63,89,920
+
+    idx = df[df[0] == "Year and Month"].index
+    df = df.iloc[idx.values[0] + 1:][:]
+    df = df.dropna(axis=1, how='all')
+    cols = [
+        "Year and Month", "Resident Population",
+        "Resident Population Plus Armed Forces Overseas", "Civilian Population",
+        "Civilian NonInstitutionalized Population"
+    ]
+    df.columns = cols
+    for col in df.columns:
+        df[col] = df[col].str.replace(",", "")
+    return df
+
+
+def _clean_txt_file(df: pd.DataFrame) -> pd.DataFrame:
+    """
+    This method cleans the dataframe loaded from a txt file format.
+    Also, Performs transformations on the data.
+
+    Args:
+        df (DataFrame) : DataFrame of txt dataset
+        scaling_factor_txt_file (int) : Scaling factor for text file
+
+    Returns:
+        df (DataFrame) : Transformed DataFrame for txt dataset.
+    """
+    # Month and Year are concatenated into a single column if they are not None
+    df['Year and Month'] = df[['Year and Month', 'Date']]\
+                                    .apply(_concat_cols, axis=1)
+    df.drop(columns=['Date'], inplace=True)
+    for col in df.columns:
+        df[col] = df[col].str.replace(",", "")
+
+    # The index numbers alotted as per where the columns are present to
+    # move the columns left
+    resident_population = 1
+    resident_population_plus_armed_forces_overseas = 2
+    civilian_population = 3
+    civilian_noninstitutionalized_population = 4
+    # Moving the row data left upto one index value.
+    # As the text file has (census) mentioned in some rows and it makes the
+    # other column's data shift by one place, we need to shift it back to the
+    # original place.
+    idx = df[df['Resident Population'] == "(census)"].index
+    df.iloc[idx, resident_population] = df.iloc[idx][
+        "Resident Population Plus Armed Forces Overseas"]
+    df.iloc[idx, resident_population_plus_armed_forces_overseas] = df.iloc[idx][
+        "Civilian Population"]
+    df.iloc[idx, civilian_population] = df.iloc[idx][
+        "Civilian NonInstitutionalized Population"]
+    df.iloc[idx, civilian_noninstitutionalized_population] = np.NAN
+    return df
+
+
+def _download(download_path: str, file_urls: list) -> None:
```

Comment:
since this function is invoked from the test file, it can be without the '_' prefix.
https://google.github.io/styleguide/pyguide.html#3164-guidelines-derived-from-guidos-recommendations

Reply:
Change made